byron/crypto/src/Cardano/Crypto/Signing/Redeem/Compact.hs

```diff
@@ -70,7 +70,7 @@ instance ToCBOR CompactRedeemVerificationKey where
 
 instance FromCBOR CompactRedeemVerificationKey where
   fromCBOR = do
-    enforceSize "CompactRedeemVerificationKey" 1
+    enforceSize "CompactRedeemVerificationKey" 4
```

Comment:
Looks like there was a bug. This was a recent bug, so I don't think we need to worry too much about this bug surfacing in the wild.

Reply:
good catch!